pom.xml

```diff
@@ -62,7 +62,7 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>groovy-sandbox</artifactId>
-      <version>1.30</version>
+      <version>1.31</version>
```

Comment:
BTW I think it would be a good investment of time to inline the lib into this plugin repo.